README.md

```diff
@@ -1 +1,46 @@
-hello-microverse
+Hello Microverse
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+Built With
+* HTML, CSS
+
+Live Demo (if available)
+Live Demo Link
+
+Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+
+Setup
+
+Install
+
+Usage
+
+Run tests
+
+Deployment
```

Comment:
- [ ] Kindly edit the empty sections with information about your project or simply remove all the necessary sections from your README file in order to make it more professional 👍